src/api/config/initializers/airbrake.rb

```diff
@@ -95,6 +95,10 @@ def ignore_by_backend_400_message?(message)
   messages_to_ignore.each do |ignored_error_message|
     return true if message.include?(ignored_error_message)
   end
+  messages_to_ignore_re = ['<summary>[^<>]*: conflict in file [^<>]*</summary>'].freeze
```

Comment:
What about merging the previous `messages_to_ignore` list and this one? I have the impression that maybe this regexp would cover some of the previous errors also...

Reply:
It's already included above: `message.include?('<summary>conflict in file')` right?

Reply:
I omitted the `: ` so that now the Regexp matches both.

Reply:
I don't get it, the message changed or what happened?

Reply:
yes the part with `prj/pkg: ` got added.

Reply:
See [here](https://github.com/openSUSE/open-build-service/pull/14444/files#diff-567a729df5a343260e81707d6c050fb07cfd16878914491a892f24e866f3161aR14-R15) to get more details